### DISCUSSION TITLE
Basic installation questions regarding Standard HA database, Influxdb, Grafana

### DISCUSSION THREAD

**Discussion by hartmood** [VictoriaMetrics-Community/homeassistant-addon-victoriametrics#22] [Q&A] [answered]
I followed the installation instructions, and it is running. At least I can see via Samba that data is constantly being written to the "small" folder. I still need to understand how Standard HA database, Influxdb, Victoriametrics and Grafana play together. Consequently, I have these questions:

1. is the data kept redundant, or is the data now only written to VM?

2. With Prometheus as the source, I got a connection via http://ipaddress_of_raspi:8428, the user "home assistant", and my password; at least no error message came up. Unfortunately, I can not display any data. What is the setting for configuring the InfluxDB source in Grafana, or can anybody provide a more detailed description of the settings, as step 6 in the documentation could be more meaningful?

3. With the active Prometheus connection, how can I use Grafana dashboard templates that usually refer to InfluxDB? Is there a simple way to use them, or do I have to change the source in the JSON file?

4. Can I use templates for this setup not to reinvent the wheel? Where can I find them?

**Comment by fuslwusl** [comment, marked as answer]
1. You have to configure the influxdb integration as described in the documentation "Sending data to Victoria Metrics". There you can define what you want to store in Victoria Metrics. If entities are added to recorder and influxdb they will be stored in both databases. Recorder usually purges data after about 10 days and is required for history display in the homeassistant ui. Victoria Metrics can be used for long term storage and visualization with Grafana.
2. I recommend using the influxdb integration if you don't need external sources.
3. Works similar to influxdb data but i recommend influxdb because everything is in the documentation. You can add a good documentation here if you like.
4. I'm not sure what templates you mean but the documentation should contain everything you need https://github.com/fuslwusl/homeassistant-addon-victoriametrics/blob/main/victoria-metrics/DOCS.md  If something is missing just add it here as pull request.
